### PR TITLE
Itsmekori/eng 322 broken connect wallet text size

### PR DESCRIPTION
**Pull request by IvySwing**

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -6,7 +6,12 @@ const withBundleAnalyzer = require("@next/bundle-analyzer")({
 const nextConfig = {
   reactStrictMode: true,
   images: {
-    domains: ["images.unsplash.com", "tailwindui.com", "res.cloudinary.com"],
+    domains: [
+      "images.unsplash.com",
+      "tailwindui.com",
+      "res.cloudinary.com",
+      "imgs.search.brave.com",
```
Comment (illepic) [inline next.config.js:14]: What's going on here?

File: components/wallet-button.tsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+                  <Button
+                    text="Wrong Network"
+                    onClick={openChainModal}
+                    type="button"
+                  />
+                );
+              }
+              return (
+                <div style={{ display: "flex", gap: 12 }}>
+                  <button
```
Comment (illepic) [inline components/wallet-button.tsx]: Can we instead use our existing `Button` component that we put a lot of work into already, that is already styled and takes in a prop function?

File: next.config.js (hunk 1 above)
Comment (illepic) [reply next.config.js:14]: Ah, these are the chain icons? Toss a comment in here that says that.

**Review by illepic — CHANGES_REQUESTED** [review]

**Comment by illepic** [comment]
Oops, didn't see this was draft and added a review. Apologies.

File: components/wallet-button.tsx (hunk 2 above)
Comment (IvySwing) [reply components/wallet-button.tsx]: I left you a message explaining why I went in this particular direction.  The restraint in Rainbow was a big factor.

**Comment by IvySwing** [comment]
@illepic Also, I need to update the address and name button after wallet connection with a new icon. I will ping you when it's complete

**Review by illepic — CHANGES_REQUESTED** [review]
Putting this in "request changes" state since we have a few things left to do that we talked about on friday.

**Review by illepic — APPROVED** [review]

**Pull request merged**